pset-functions.js

```diff
@@ -207,7 +298,33 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 5 - bonus:')
 
 // Add your code below this line
-
+// create a function called wordCount with parameter called phrase
+function wordCount(phrase) {
+  const words = phrase.split(' ')
+  const wordCount = {}
+
+  words.forEach((word) => {
+    // check if word exists as a key in wordCount object
+    // if it does not, then add word as key and value as 1
+    // if word already exists, then increment value by 1
+    if (wordCount[word] === undefined) {
```

Comment:
Matt, this solution is brilliant! You make great use of how objects work and inserted a key/value pair when it is not currently available.

Overall, really great job getting this homework completed. Your code has proper indentation, you made great use of pseudo code for the steps you take, and your logic is sound. Keep up the great work!